Library/Tracking/KSRAnalyticsTests.swift

```diff
@@ -1474,7 +1474,7 @@ final class KSRAnalyticsTests: TestCase {
     XCTAssertEqual("sign_up", dataLakeClient.properties.last?["context_page"] as? String)
     XCTAssertEqual("sign_up", segmentClient.properties.last?["context_page"] as? String)
 
-    ksrAnalytics.trackThanksPageViewed(project: .template, reward: .template, checkoutData: nil)
+    ksrAnalytics.trackThanksPageViewed(project: .template, reward: .template, checkoutData: nil, typeContext: .pledge(.newPledge))
```

Comment:
Related to the comment above. After re-factoring this method, you won't need to hard-code the typeContext argument.